diff --git a/snake.asm b/snake.asm
index 4353620..4863482 100644
--- a/snake.asm
+++ b/snake.asm
@@ -91,6 +91,20 @@ start:
 .food:
 	inc bp
 	inc bp
+	mov di, 0x230
+	add word [score], 0x1
+	mov ax, [score]
+	mov bl, 0xA
+.print_score:
+	div bl
+	xchg al, ah
+	add al, '0'
+	stosb
+	sub di, 0x3
+	mov al, ah
+	xor ah, ah
+	or al, al
+	jnz .print_score
 	call print_food
 .done:
 	pop di
